etc/adminhtml/system.xml

```diff
@@ -33,40 +36,49 @@
                     <depends>
                         <field id="enable_pending_orders_cron">1</field>
                     </depends>
+                    <frontend_model>Razorpay\Magento\Model\FormFieldTrack</frontend_model>
                     <validate>required-entry validate-digits validate-not-negative-number validate-digits-range digits-range-20-43200</validate>
                 </field>
 
                 <field id="title" translate="label" type="text" sortOrder="1" showInDefault="1" showInWebsite="1" showInStore="1">
                     <label>Title</label>
+                    <frontend_model>Razorpay\Magento\Model\FormFieldTrack</frontend_model>
                     <config_path>payment/razorpay/title</config_path>
                 </field>
                 <field id="merchant_name_override" translate="label" type="text" sortOrder="1" showInDefault="1" showInWebsite="1" showInStore="1">
                     <label>Merchant Name</label>
+                    <frontend_model>Razorpay\Magento\Model\FormFieldTrack</frontend_model>
                     <config_path>payment/razorpay/merchant_name_override</config_path>
                 </field>
                 <field id="key_id" translate="label" type="text" sortOrder="3" showInDefault="1" showInWebsite="1" showInStore="0">
                     <label>API Key</label>
                     <comment>**Required (Enter test key ID for testing)**</comment>
                     <config_path>payment/razorpay/key_id</config_path>
+                    <frontend_model>Razorpay\Magento\Model\FormFieldTrack</frontend_model>
                     <validate>required-entry</validate>
                 </field>
                 <field id="key_secret" translate="label" type="text" sortOrder="4" showInDefault="1" showInWebsite="1" showInStore="0">
                     <label>API Key Secret</label>
                     <comment>**Required (Enter test key secret for testing)**</comment>
+                    <frontend_model>Razorpay\Magento\Model\FormFieldTrack</frontend_model>
                     <validate>required-entry</validate>
                     <config_path>payment/razorpay/key_secret</config_path>
                 </field>
                 <field id="order_status" translate="label" type="select" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="0">
                     <label>New Order Status</label>
                     <source_model>Magento\Sales\Model\Config\Source\Order\Status\NewStatus</source_model>
+                    <frontend_model>Razorpay\Magento\Model\FormFieldTrack</frontend_model>
                     <validate>required-entry</validate>
                     <config_path>payment/razorpay/order_status</config_path>
+                    <!-- <frontend_model>Razorpay\Magento\Model\FormFieldTrack</frontend_model> -->
```

Comment:
@nikhilsharma1100 Please remove this line of code as this is not in use.

Reply:
done